### ISSUE TITLE
paraStyle and blockTableStyle names are not handled separate

### ISSUE THREAD

**Issue by joeUser555** [zopefoundation/z3c.rml#108]
### What I did:

I defined a paraStyle and a blockTableStyle with an identical name.

### What I expect to happen:

In a para element the paraStyle name referenced should give the paraStyle with this name.

### What actually happened:

The blockTableStyle with the identical name got returned and caused an exception:
```
AttributeError: 
paragraph text '<para>paraStyle Helvetica 12pt</para>' caused exception 'TableStyle' object has no attribute 'name'
```
[style-name-conflict.rml.txt](https://github.com/zopefoundation/z3c.rml/files/10798016/style-name-conflict.rml.txt)
### What version of Python and Zope/Addons I am using:

Ubuntu 22.04, Python 3.10, zope-event-4.6

**Comment by regebro** [comment]
Yes, this is correct, and as far as I can tell so does reportlab.

**Comment by joeUser555** [comment]
If paraStyle names and blockTableStyle names share the same namespace the parser should yield a warning or better an error if the style name is use in both categories in one document.

**Comment by regebro** [comment]
Warning if you redefine a style does make sense.
